test_sort_function.py

```
@@ -0,0 +1,74 @@
+    "Wonder Woman Dead Earth TPB (December 2020)": "Wonder Woman - Dead Earth-000.jpg"
+}
+
+IMG_EXTS = {".jpg", ".jpeg", ".png", ".webp"}
+
+
+def iter_archives(root: Path) -> list[Path]:
+    return sorted(p for p in root.rglob("*") if p.suffix.lower() in (".cbz", ".zip"))
+
+
+def make_test_method(archive_path: Path):
+    def test(self):
```

Ruff (0.11.9)

Missing return type annotation for private function `test`

Add return type annotation: `None`

(ANN202)